internal/k8s-engine/validate/action.go

```diff
@@ -11,16 +11,19 @@ import (
 	"github.com/pkg/errors"
 )
 
+// ActionValidator provides functionality to statically validate Action definition.
 type ActionValidator struct {
 	wfCli *wfclientset.Clientset
 }
 
+// NewActionValidator returns a new ActionValidator instance.
 func NewActionValidator(wfCli *wfclientset.Clientset) *ActionValidator {
 	return &ActionValidator{
 		wfCli: wfCli,
 	}
 }
 
+// Validate a given Action definition.
```

Comment:
Suggestion: While it is a valid sentence, I would recommend unifying that with the rest of the comments 🙂 For example with `// Update updates current Capact Policy configuration with a given input.`

```suggestion
// Validate validates a given Action definition.
```